internal/cmd/scraper/scraper.go

```diff
@@ -296,7 +296,7 @@ func Run(cfg *Config) error {
 		case "infra-sdk":
 			specs, err := integration.LoadSpecFiles(cfg.DefinitionFilesPath)
 			if err != nil {
-				return err
+				logrus.Errorf("error loading definition files: %s", err)
```

Comment:
Do we want to carry on even if the file is not found/not loaded?

Reply:
i don't see why not. When the scraped target has no matching on the definitions the metrics are added to the host entity. Also this is checking that the folder is there

Reply:
Yes that is right 👍